src/components/signature/signature.tsx

```diff
@@ -0,0 +1,44 @@
+import differenceInCalendarDays from 'date-fns/differenceInCalendarDays';
```

Comment:
our package.json misses the explicit dependency on `date-fns`. Please add it if you need this helper.